libs/blocks/modal/modal.css

```
@@ -332,6 +333,10 @@
     border-radius: 0;
   }
 
+  .dialog-modal:not(.tall-video, .three-in-one, .commerce-frame) {
```

I think `.upgrade-flow-modal` should be added as well because of https://github.com/adobecom/milo/blob/stage/libs/blocks/modal/modal.css#L30
```suggestion
  .dialog-modal:not(.tall-video, .three-in-one, .commerce-frame, .upgrade-flow-modal) {
```